php-frontend/src/test/java/org/sonar/plugins/php/api/tests/PHPCheckVerifierTest.java

```diff
@@ -47,7 +47,7 @@ public void legacy_file_issue() throws Exception {
       new PHPVisitorCheck() {
```

Comment:
as legacy issue is removed, should we remove also the related unit tests ?

Reply:
Fair point, thank you!